app/views/notifications/_nav_menu.html.erb

```diff
@@ -1,72 +1,42 @@
-      <div class="pt-6">
-        <h3 class="crayons-subtitle-3 mb-4 pl-2"><%= org.name %></h3>
-        <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:filter].to_s.casecmp("org").zero? && params[:org_id].to_i == org.id %>" href="/notifications/org/<%= org.id %>">
-          All
-        </a>
-        <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].to_i == org.id %>" href="/notifications/comments/<%= org.id %>">
-          Comments
-        </a>
-      </div>
-    <% end %>
-  <% end %>
-</nav>
-
-<nav class="block m:hidden px-4" aria-label="Notifications">
-  <% if @organizations.present? %>
-    <select id="notifications-filter__select" class="crayons-select mb-2">
+    <select id="notifications-filter__select" class="crayons-select mb-4" aria-label="Filter by account...">
       <option value="/notifications"><%= @user.name %></option>
       <% @organizations.each do |org| %>
         <option value="/notifications/org/<%= org.id %>" <%= "selected" if params[:org_id].to_i == org.id %>><%= org.name %></option>
       <% end %>
     </select>
   <% end %>
 
-  <div class="crayons-tabs <%= "hidden" if params[:org_id].present? %>">
-    <ul class="crayons-tabs__list">
-      <li>
-        <a data-text="All" class="crayons-tabs__item <%= "crayons-tabs__item--current" if params[:filter].blank? %>" href="/notifications" aria-current="<%= "page" if params[:filter].blank? %>">
-          All
-        </a>
-      </li>
-      <li>
-        <a data-text="Comments" class="crayons-tabs__item <%= "crayons-tabs__item--current" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].blank? %>" href="/notifications/comments" aria-current="<%= "page" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].blank? %>">
-          Comments
-        </a>
-      </li>
-      <li>
-        <a data-text="Posts" class="crayons-tabs__item <%= "crayons-tabs__item--current" if params[:filter].to_s.casecmp("posts").zero? && params[:org_id].blank? %>" href="/notifications/posts" aria-current="<%= "page" if params[:filter].to_s.casecmp("posts").zero? && params[:org_id].blank? %>">
-          Posts
-        </a>
-      </li>
-    </ul>
-  </div>
+  <ul class="crayons-navigation <%= "hidden" if params[:org_id].present? %>">
+    <li>
+      <a data-text="All" class="crayons-navigation__item <%= "crayons-navigation__item--current" if params[:filter].blank? %>" href="/notifications" aria-current="<%= "page" if params[:filter].blank? %>">
+        All
+      </a>
+    </li>
+    <li>
+      <a data-text="Comments" class="crayons-navigation__item <%= "crayons-navigation__item--current" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].blank? %>"
+        href="/notifications/comments" aria-current="<%= "page" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].blank? %>">
```

Comment:
We could do some CSS magic so that we don't always have to have the same block of code to decide if something is both `crayons-navigation__item--current` and `aria-current="page"` . 

We could use the CSS selector `.crayons-navigation__item[aria-current="page"]` and apply the 'current' styles 🌈  Might save a fair chunk of repeated code we've had around for a while, and a fringe benefit of helping us to remember to set that `aria-current` property for future nav tab groups

Reply:
Great idea!